tests/index.ts

```diff
@@ -0,0 +1,24 @@
+import Ship from "../lib/models/ships/model.ship";
+import dotenv from "dotenv";
+import getResolver from "../lib/getResolver";
+import { is } from "uvu/assert";
+import path from "path";
+import { suite } from "uvu";
+
+const $: uvu.Test<Record<string, any>> = suite("sample test");
+
+// Setup process environment variables for uvu
+const config = dotenv.config({ path: path.resolve(process.cwd(), ".env.local") });
+process.env.GITHUB_REPO = config.parsed.GITHUB_REPO;
+process.env.GITHUB_TOKEN = config.parsed.GITHUB_TOKEN;
+
+$("should do some math", async () => {
```

Comment:
This looks unnecessary.

Reply:
Could update this part in [`60ecf5c`](https://github.com/LeNitrous/azur-lane-api/pull/10/commits/60ecf5cc220ae14a5e570715134a823e05d0ac5a), the math test was mainly for demonstration, as previously mentionned in the issue thread. I agree, the two above lines are unnecessary.